project-management/src/main/java/life/qbic/projectmanagement/domain/model/experiment/Experiment.java

```diff
@@ -84,6 +94,24 @@ public String getName() {
     return name;
   }
 
+  public String getSpeciesIconName() {
+    return speciesIconName;
+  }
+
+  public String getSpecimenIconName() {
+    return specimenIconName;
+  }
+
+  public String getAnalyteIconName() {
+    return analyteIconName;
+  }
+
+  public void setIconNames(String speciesIconName, String specimenIconName, String analyteIconName) {
+    this.speciesIconName = speciesIconName;
+    this.specimenIconName = specimenIconName;
+    this.analyteIconName = analyteIconName;
+  }
```

Comment:
some sanity checks might be useful here (null, empty, allowed value). Maybe think about providing enums.